css-content/Overview.bs

```diff
@@ -1087,7 +1088,7 @@ The ''string()'' function</h4>
 The ''content()'' function</h4>
 
 	<pre class="prod">
-		<dfn>content()</dfn> = content( [text | before | after | first-letter | marker ]? )
+		<dfn><<content()>></dfn> = content( [ text | before | after | first-letter | marker ] )
```

Comment:
same.

Also, don't remove the `?`, that's a significant and important bit. (It allows the "If no value is specified" bit a few lines later down.)